gallery/visualtests/nxVulnerabilityIndicator.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { setupBrowser } = require('./testUtils');
+
+describe('NxVulnerabilityIndicator', function() {
+  const { simpleTest, a11yTest } = setupBrowser('#/pages/Vulnerability Indicator');
+
+  const vulnerabilityIndicatorsListExampleSelector = '#nx-vulnerability-indicator-list-example .nx-list',
+      vulnerabilityIndicatorsTableExampleSelector = '#nx-vulnerability-indicator-table-example .nx-table',
+      vulnerabilityIndicatorsSimpleExampleSelector = '#nx-vulnerability-indicator-simple-example .gallery-example-live',
+      vulnerabilityIndicatorsScoreExampleSelector =
+        '#nx-vulnerability-indicator-score-example .gallery-example-live';
+
+  it('looks right', simpleTest(vulnerabilityIndicatorsSimpleExampleSelector));
+  it('looks right in a list', simpleTest(vulnerabilityIndicatorsListExampleSelector));
+  it('looks right in a table', simpleTest(vulnerabilityIndicatorsTableExampleSelector));
+  it(
+      'looks right when score prop is provided',
+      simpleTest(vulnerabilityIndicatorsScoreExampleSelector)
+  );
```

Comment:
One line

Reply:
There've been a lot of weird line breaks in these PRs.  Considering changing your text editor's settings.

Reply:
Thank for pointing that out, fixed in: [a32275b](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/a32275bd3f95f5dc52a205829c697b3fd709a186)